kitsune/search/v2/search.py

```diff
@@ -159,6 +159,42 @@ def make_result(self, hit):
         }
 
 
+class ProfileSearch(SumoSearch):
+    """Search over User Profiles."""
+
+    def get_index(self):
+        return ProfileDocument.Index.read_alias
+
+    def get_fields(self):
+        return ["username", "name"]
+
+    def get_highlight_fields(self):
+        return []
+
+    def get_filter(self, **kwargs):
+        return DSLQ(
+            "boosting",
+            positive=self.build_query(**kwargs),
+            negative=DSLQ(
+                "bool",
+                must_not=DSLQ("terms", group_ids=kwargs.get("group_ids", [])),
```

Comment:
The `group_ids` kwarg ought to be named `boost_group_ids` or similar

Reply:
imo `boost_group_ids` is a bit misleading. Basically we are matching against specific group IDs here and we are not boosting anything related to IDs but rather the results of the filtering

Reply:
`group_ids` alone implies to me that we're filtering on that list - users not in those groups won't appear at all - which isn't what's happening, but consider this a nit